Src/ResistanceOperationOverhaul/Classes/UITLE_LadderModeMenu_Override.uc

```diff
@@ -1302,6 +1302,12 @@ simulated function CreateNewLadder(int LadderDifficulty, bool NarrativesOn)
 	AddPlayerState = class'XComGameState_Player'.static.CreatePlayer(TacticalStartState, eTeam_Resistance);
 	BattleDataState.PlayerTurnOrder.AddItem(AddPlayerState.GetReference());
 
+	AddPlayerState = class'XComGameState_Player'.static.CreatePlayer(TacticalStartState, eTeam_One);
```

Comment:
eTEam_One and eTeam_Two are additional teams (Originally MP teams) Added by the highlander that various mods (Namely the raider factions and the Hive mod) use to assign their units to to be enemies to both ADVENT and XCOM without using eTeam_TheLost. In future when I or someone else finds a way to potentially add sit-reps to each mission, It would allow raider incursions/the Hive to work properly instead of spawning on an invalid team.